### PR TITLE
Suggested text enhancements to make sure important info has better visibility

### PR DESCRIPTION
**Pull request by softchris**
- create more descriptive header
- added more emphasis on certain places

### CODE REVIEW

**Review by peacing — APPROVED** [review]

File: docs/index.md [hunk 1]
```diff
@@ -6,180 +6,36 @@ nav_order: 0
...
-The open source lakeFS project supports AWS S3, Azure Blob Storage and Google Cloud Storage (GCS) as its underlying storage service. It is API compatible with S3 and integrates seamlessly with popular data frameworks such as Spark, Hive, dbt, Trino, and many others.
+In this way, lakeFS brings a unique combination of performance and manageability to data lakes. To learn more about applying Git principles to data, [see here](https://lakefs.io/how-to-manage-your-data-the-way-you-manage-your-code/).
 
-{: .pb-5 }
+## Supports the major cloud providers
 
+The open source lakeFS project supports AWS S3, Azure Blob Storage and Google Cloud Storage (GCS) as its underlying storage service. It is API compatible with S3 and integrates seamlessly with popular data frameworks such as Spark, Hive, dbt, Trino, and many others.
 
-## New! lakeFS Playground
+## NEW lakeFS Playground
```
Comment (johnnyaug) [inline docs/index.md:23]:
```suggestion
## lakeFS Playground
```

File: docs/index.md [hunk 2]
```diff
@@ -6,180 +6,36 @@ nav_order: 0
...
-
-Binary packages are available for Linux/macOS/Windows on [GitHub Releases](https://github.com/treeverse/lakeFS/releases){: target="_blank" }
-
-#### Docker Images
-
-Official Docker images are available at [https://hub.docker.com/r/treeverse/lakefs](https://hub.docker.com/r/treeverse/lakefs){: target="_blank" }
-
+> df = spark.read.parquet("s3a://my-repo/main-branch/collections/foo/")
 
-### Next steps
```
Comment (johnnyaug) [inline docs/index.md:183]: Why remove the next steps?

File: docs/why-lakefs.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
+---
+layout: default
+title: Why is lakeFS the data solution you’ve been missing?
```
Comment (johnnyaug) [inline docs/why-lakefs.md:3]: Why is this a top level element? This looks strange:
image

File: docs/index.md [hunk 4]
```diff
@@ -6,180 +6,36 @@ nav_order: 0
...
-
-#### Binary Releases
-
-Binary packages are available for Linux/macOS/Windows on [GitHub Releases](https://github.com/treeverse/lakeFS/releases){: target="_blank" }
-
-#### Docker Images
-
-Official Docker images are available at [https://hub.docker.com/r/treeverse/lakefs](https://hub.docker.com/r/treeverse/lakefs){: target="_blank" }
-
+> df = spark.read.parquet("s3a://my-repo/main-branch/collections/foo/")
```
Comment (johnnyaug) [inline docs/index.md:38]: Why not render this as a code block?

File: docs/index.md [hunk 5]
```diff
@@ -6,180 +6,36 @@ nav_order: 0
...
+lakeFS maintains compatibility with the S3 API to minimize adoption friction. Use it as a drop-in replacement for S3 from the perspective of any tool interacting with a data lake.
 
 For example, take the common operation of reading a collection of data from object storage into a Spark DataFrame. For data outside a lakeFS repo, the code will look like:
 
-```py
-df = spark.read.parquet("s3a://my-bucket/collections/foo/")
-```
-
+> df = spark.read.parquet("s3a://my-bucket/collections/foo/")
 After adding the data collections in my-bucket to a repository, the same operation becomes:
```
Comment (johnnyaug) [inline docs/index.md:36]: This is placed in the same line as the code above

File: docs/index.md [hunk 6]
```diff
@@ -6,180 +6,36 @@ nav_order: 0
...
-any tool interacting with a data lake.
+lakeFS maintains compatibility with the S3 API to minimize adoption friction. Use it as a drop-in replacement for S3 from the perspective of any tool interacting with a data lake.
 
 For example, take the common operation of reading a collection of data from object storage into a Spark DataFrame. For data outside a lakeFS repo, the code will look like:
 
-```py
-df = spark.read.parquet("s3a://my-bucket/collections/foo/")
-```
-
+> df = spark.read.parquet("s3a://my-bucket/collections/foo/")
```
Comment (johnnyaug) [inline docs/index.md:35]: Why not a code block?

File: docs/index.md [hunk 7]
```diff
@@ -6,180 +6,36 @@ nav_order: 0
...
-Experience lakeFS first hand with your own isolated environment.
-You can easily integrate it with your existing tools, and feel lakeFS in action in an environment similar to your own.
+Experience lakeFS first hand with your own isolated environment. You can easily integrate it with your existing tools, and feel lakeFS in action in an environment similar to your own.
 
-<p style="margin-top: 40px;">
-    <a class="btn btn-green" href="https://demo.lakefs.io/" target="_blank">
-        Try lakeFS now without installing
-    </a>
-</p>
+> [Try lakeFS now without installing](https://demo.lakefs.io/)
```
Comment (johnnyaug) [inline docs/index.md:27]: This used to be a button - it's now a link. Is this on purpose?

File: docs/why-lakefs.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+- It maintains high performance over data lakes of any size
+- It includes configurable garbage collection capabilities
+- It is highly available and production ready
+lakeFS integration into data lake
+
+<img src="{{ site.baseurl }}/assets/img/lakeFS_integration.png" alt="lakeFS integration into data lake" width="60%" height="60%" />
+
+## Start using lakeFS - available downloads
+
+- BINARY RELEASES. Binary packages are available for Linux/macOS/Windows on [See GitHub Releases](https://github.com/treeverse/lakeFS/releases)
```
Comment (johnnyaug) [inline docs/why-lakefs.md:88]: I wouldn't use all caps anywhere

Comment (johnnyaug) [inline docs/why-lakefs.md:88]:
```suggestion
- BINARY RELEASES. Binary packages are available on our [GitHub Releases](https://github.com/treeverse/lakeFS/releases) page.
```

File: docs/why-lakefs.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+
+- DOCKER IMAGES. Official Docker images are available at <https://hub.docker.com/r/treeverse/lakefs>
```
Comment (johnnyaug) [inline docs/why-lakefs.md:90]: Why are docker images and binaries are part of this page? It doesn't seem related to the title.

**Review by johnnyaug — CHANGES_REQUESTED** [review]
I don't understand the logic behind this split - it doesn't seem coherent.

Also, please render the docs locally before submitting a PR. [This guide](https://docs.google.com/document/d/1vybFO6Ib8AZ9kRMs_Sb4W0gkt2M6Tpu57pnsH9I_O3g) explains how to do it.

**Comment by softchris** [comment]
> I don't understand the logic behind this split - it doesn't seem coherent.
> 
> Also, please render the docs locally before submitting a PR. [This guide](https://docs.google.com/document/d/1vybFO6Ib8AZ9kRMs_Sb4W0gkt2M6Tpu57pnsH9I_O3g) explains how to do it.

the reason was because there's too much happening in the doc currently imo.. It's hard to digest. But that's just my opinion.

File: docs/index.md (hunk 6 above)
Comment (softchris) [reply docs/index.md:35]: this is not a tutorial page. It's an information page. Code blocks makes sense in a tutorial.

File: docs/index.md (hunk 4 above)
Comment (softchris) [reply docs/index.md:38]: this is not a tutorial page. It's an information page. Code blocks makes sense in a tutorial.

File: docs/index.md (hunk 2 above)
Comment (softchris) [reply docs/index.md:183]: did I remove it completely or move it into another doc?

File: docs/why-lakefs.md (hunk 3 above)
Comment (softchris) [reply docs/why-lakefs.md:3]: The "what" and the "why" is the most important things you want to know about any one topic. They should be really easy to find in a doc, either as standalone elements or inside of the same doc.

Personally I think it's important to think about snackable content, not too long articles, that's the reason why I would raise this PR, I felt the existing article was a bit compact.

I see your point on rendering but what about this approach?

lakeFS/
  Why
  What

@adipolak wdyt?

File: docs/why-lakefs.md (hunk 8 above)
Comment (softchris) [reply docs/why-lakefs.md:88]: > I wouldn't use all caps anywhere

you were in your h4 headers which I took away and replaced with h3 @adipolak

**Comment by arielshaqed** [comment]
Sorry, no useful input from me on this one.

**Comment by johnnyaug** [comment]
Inactive, closing.
